lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -364,7 +414,7 @@ def _calculate_power_spectral_density(self, f, fc, diffusion_constant, *filter_p
         physical_spectrum = self._passive_power_spectrum_model(f, fc, diffusion_constant)
         return physical_spectrum * self._filter(f, *filter_params)
 
-    def __call__(self, f, fc, diffusion_constant, *filter_params):
+    def __call__(self, f, fc, diffusion_constant, *filter_params) -> np.ndarray:
```

Comment:
so far we've actually been using the proper type annotation for properties and for methods putting a `Returns` section in the docstring. Not saying we have to keep going this way but at some point we should make things consistent